101-webapp-privateendpoint-vnet-injection/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetwork_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Name of the VNet"
+      }
+    },
+    "serverFarm_name": {
+      "type": "String",
+      "defaultValue": "ServerFarm1",
+      "metadata": {
+        "description": "Name of the Web Farm"
+      }
+    },
+    "site1_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Web App 1 name must be unique DNS name worldwide"
+      }
+    },
+    "site2_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Web App 2 name must be unique DNS name worldwide"
+      }
+    },
+    "virtualNetwork_CIDR": {
+      "type": "String",
+      "defaultValue": "10.200.0.0/16",
+      "metadata": {
+        "description": "CIDR of your VNet"
+      }
+    },
+    "subnet1_name": {
+      "type": "String",
+      "defaultValue": "Subnet1",
+      "metadata": {
+        "description": "Name of the subnet"
+      }
+    },
+    "subnet2_name": {
+      "type": "String",
+      "defaultValue": "Subnet2",
+      "metadata": {
+        "description": "Name of the subnet"
+      }
+    },
+    "subnet1_CIDR": {
+      "type": "String",
+      "defaultValue": "10.200.1.0/24",
+      "metadata": {
+        "description": "CIDR of your subnet"
+      }
+    },
+    "subnet2_CIDR": {
+      "type": "String",
+      "defaultValue": "10.200.2.0/24",
+      "metadata": {
+        "description": "CIDR of your subnet"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "SKU_name": {
+      "type": "String",
+      "defaultValue": "P1v2",
+      "metadata": {
+        "description": "SKU name, must be minimum P1v2"

```

Comment:
add allowedValues

Reply:
yes that's why I explained that the SKU must be minimum P1v2

Reply:
If that's the only allowedValue, move this to a variable.  If other skus are allowed, specify that via the allowedValues property on the parameter definition.

Reply:
done